docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,680 @@
++
+A message displays that the RHOAS collection was installed successfully to the following path: `home/<usr>/.ansible/collections/ansible_collections/rhoas/rhoas`. All modules for the collection can be found in this location.
++
+When installed, you can reference collection content by the fully qualified collection name (FQCN) of the collection. The command in a task needs to include the FQCN of the module so it always points to the intended collection. For example, the FQCN for the `create_kafka` command is
+`rhoas.rhoas.create_kafka`.
++
+[NOTE]
+====
+To find documentation for each module, open a terminal window, enter `ansible-doc rhoas.rhoas._<module_name>_` and replace _<module_name>_ with the name of the module in the terminal. The following example outputs the documentation for the `create_kafka` module. The documentation shows required parameters (denoted by `=`) and optional parameters (denoted by `-`) for the module, as well as examples of how to use the module and the output.
+
+[source, shell]
+----
+$ ansible-doc rhoas.rhoas.create_kafka
+----
+====
++
+
+
+[id="proc-using-rhoas-ansible-collection_{context}"]
+
+=== Using the {product-long-rhoas} Ansible collection
+
+[role="_abstract"]
+You can use the OpenShift Application Services (RHOAS) https://galaxy.ansible.com/rhoas/rhoas[Ansible collection] to fully manage your Kafka environment with Ansible. You can create, view, configure, and delete the following {product-long-kafka} resources:
+
+* Kafka instance
+* Service accounts
+* Topics
+* ACLS
+
+The procedure in this section shows how to create and run an example playbook. The example playbook includes modules for creating a Kafka instance, topic, and service account, and assigning permissions to the service account. You also configure modules to delete the resources that you previously created.
+
+.Prerequisites
+* You have a Red Hat account.
+* You have an offline token used to authenticate the Ansible modules with the {product-long-rhoas} API.
+
+[NOTE]
+The offline token is a refresh token with no expiry and can be used by non-interactive processes to provide an access token for {product-long-rhoas} to authenticate the user. The token is an OpenShift offline token and you can find it at https://cloud.redhat.com/openshift/token.
+
+
+.Procedure
+. The collection requires the following two variables to work:
+_API_BASE_HOST_:: The base host for the API. This is the base URL for the API. For example, https://api.openshift.com.
+_SSO_BASE_HOST_:: The base host for the SSO. This is the base URL for the SSO. For example, https://sso.redhat.com/auth/realms/redhat-external.
+You can define the environment variables shown in an environment variable (`.env`) file and save the file in the root directory. If you don't explicitly define these environment variables, the collection uses the URLs shown in the examples.
```

Comment:
These lines need to read more like instructions:

```
. In your IDE, create a new enviornment variables (`.env`) file.
. In the `.env` file, add the following variables required by the Ansible collection:
+
`API_BASE_HOST`:: The base host for the API. This is the base URL for the API. For example, `\https://api.openshift.com`.
`SSO_BASE_HOST`:: The base host for the SSO. This is the base URL for the SSO. For example, `\https://sso.redhat.com/auth/realms/redhat-external`.
. Save the file in the root directory of the Ansible collection.
+
NOTE: If you do not explicitly the environment variables, the collection uses the URLs shown in the preceding step.
```

Reply:
A couple of other comments here:

- We don't want the variable names italicized, so I replaced the underscores with backticks.
- We don't want either of the URLs to appear in the output as clickable hyperlinks, so observe the formatting I added to avoid this.
- In the descriptions, I would replace 'host' with 'URL'.
- Per a comment I left somewhere else, we should define 'SSO' before using it:  "The base URL for the Red Hat single sign-on (SSO) service."